packages/babel-core/register.js

```diff
@@ -1,3 +1,3 @@
 /* eslint max-len: 0 */
 // TODO: eventually deprecate this console.trace("use the `babel-register` package instead of `babel-core/register`");
-module.exports = require("babel-register");
+module.exports = require("babel-register").default;
```

Comment:
Yeah good point, we definitely should do that. Can you please create an issue, or a PR if you have time? Thanks.